browser.go

```diff
@@ -310,6 +310,15 @@ func (b *Browser) run(ctx context.Context) {
 	}
 }
 
+// Process returns the Chrome process object
+// Example:
+//     if process := chromedp.FromContext(ctx).Browser.Process(); process != nil {
+//         fmt.Printf("Chrome PID: %v", process.Pid)
+//     }
+func (b *Browser) Process() *os.Process {
```

Comment:
It seems that it's more natural to place this func after the func `NewBrowser`.

Reply:
Done.